_sass/_variables.scss

```diff
@@ -8,6 +8,9 @@ $color-nav: #444;
 $color-spacer: #eee;
 $color-submenu-bg: white;
 $color-bg: #fafafa;
+$color-eclipse: #3d3d3d;
+$color-gray59: #969696;
```

Comment:
However, one more step would get us to the named DimGray if you prefer.

Proofs:
https://www.color-hex.com/color/696969
https://webaim.org/resources/contrastchecker/?fcolor=696969&bcolor=f5f5f5&api

```suggestion
$color-dim-gray: #696969;
```

Reply:
![image](https://user-images.githubusercontent.com/17770407/87853471-4ab3a600-c8d8-11ea-8484-71242942ec23.png)